src/dtos/http_response.go

```diff
@@ -0,0 +1,58 @@
+package dtos
+
+import (
+	"time"
+
+	"github.com/Real-Dev-Squad/wisee-backend/src/models"
+)
+
+type ErrorResponse struct {
+	Message string      `json:"message"`
+	Detail  interface{} `json:"detail"`
```

Comment:
- will `details` be a more suitable name here? what do you think?
- what will we be sending here?

Reply:
the exact error string that can be extracted from the err object or any custom error that we want